.env.template

```diff
@@ -21,18 +21,7 @@ GITHUB_CLIENT_ID=test
 GITHUB_CLIENT_SECRET=test
 
 BUCKET_NAME=foo
-GCP_STORAGE_SERVICE_ACCOUNT='{ \
-  "type": "service_account", \
-  "project_id": "foo", \
-  "private_key_id": "aprivatekey", \
-  "private_key": "anotherprivatekey", \
-  "client_email": "aserviceaccountemail@example.com", \
-  "client_id": "aclientid", \
-  "auth_uri": "https://accounts.google.com/o/oauth2/auth", \
-  "token_uri": "https://oauth2.googleapis.com/token", \
-  "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs", \
-  "client_x509_cert_url": "acerturl" \
-}'
+GCP_STORAGE_SERVICE_ACCOUNT='{"type":"service_account","project_id":"foo","private_key_id":"aprivatekey","private_key": "anotherprivatekey","client_email": "aserviceaccountemail@example.com","client_id": "aclientid","auth_uri": "https://accounts.google.com/o/oauth2/auth","token_uri": "https://oauth2.googleapis.com/token","auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs","client_x509_cert_url": "acerturl"}'
```

Comment:
Probably better to break this up? Makes it easier to read?

Reply:
Perhaps leave a comment above it reminding whoever uses the template to remove the spaces?

Reply:
yeah, rationale was that I was a dumdum and just assumed the formatting was how it was supposed to be and disregarded the README but that's fair point, will revert

Reply:
Tbh this is probably revealing of the issue of relying on .env files. If you could get #755 done, a lot of your problems would be fixed.